microsetta_interface/templates/new_results_page.jinja2

```diff
@@ -594,6 +594,439 @@
         select.prop("disabled", false);
     }
 
+    function drawBoxPlots(top5) {
+        let mockData = top5.map(d => {
+            const relativeAbundance = parseFloat(d.relativeAbundance);
+
+            let maxRange;
+            if (relativeAbundance > 50) {
+                maxRange = 100;
+            } else if (relativeAbundance < 50 && relativeAbundance > 5) {
+                maxRange = 50;
+            } else {
+                maxRange = 5;
+            }
+
+            const minVal = 0;
+            const maxVal = maxRange;
+            const q1 = relativeAbundance * 0.25;
+            const median = relativeAbundance * 0.5;
+            const q3 = relativeAbundance * 0.75;
+
+            return {
+                genus: d.Genus || d.genus || "Unknown",
```

Comment:
```suggestion
                genus: d.Genus || d.genus || "{{ _('Unknown') }}",
```

All strings that are/could be displayed to the user need to be exposed to the I18N mechanism. Only tagging this instance, but please review all of the code for this issue.